tests/tests/optionsTests.robot

```diff
@@ -0,0 +1,95 @@
+*** Settings ***
+Documentation       Options Tests
+
+Library             SeleniumLibrary
+Library             String
+Resource            src/page_objects/keywords/common.robot
+Resource            src/page_objects/keywords/home_page.robot
+Resource            src/page_objects/keywords/ubuntu.robot
+Resource            src/test_files/test_files.resource
+Library             src/page_objects/libraries/browser_lib.py
+
+Suite Setup         Prepare Test Environment
+Test Setup          Prepare For Test
+Test Teardown       Clean After Test
+
+
+*** Test Cases ***
+Find and open Enable Debug Output
+    [Documentation]    Click Options, then toggle on/off Enable Debug output button and verify if Reload Badge appears/disappears
+    [Tags]    options
+
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+
+    Click Element    ${EnableDebugSwitch}
+    Wait Until Element Is Visible    ${ReloadBadge}
+
+    Click Element    ${EnableDebugSwitch}
+    Wait Until Element Is Not Visible    ${ReloadBadge}
+
+Find and open Exclude temporary files
+    [Documentation]    Click Options, then toggle on/off Exclude temporary files switch button and verify if Reload Badge appears/disappears
+    [Tags]    options
+
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+
+    Click Element    ${ExcludeTemporaryFilesSwitch}
+    Wait Until Element Is Visible    ${ReloadBadge}
+
+    Click Element    ${ExcludeTemporaryFilesSwitch}
+    Wait Until Element Is Not Visible    ${ReloadBadge}
+
+Find and open Output logs to a file
+    [Documentation]    Click Options, then toggle on/off Output logs to a file button and verify if Reload Badge appears/disappears
+    [Tags]    options
+
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+
+    Click Element    ${OutputLogsSwitch}
+    Wait Until Element Is Visible    ${LogsButton}
```

Comment:
It does appear that ${LogsButton} is visible while both options are selected, the element that changes is button text and icon - maybe it is worth to check that text or icon? Since right now test will work whether this option is selected or not.

Reply:
Done